GoogleMeetup/src/main/webapp/script.js

```diff
@@ -315,6 +301,168 @@ function initAutocomplete() {
 
     // Add the autocomplete container to dropdown div so menu doesn't disappear
     setTimeout(function() {
-        document.getElementById("addPerson").prepend(document.getElementsByClassName("pac-container")[0]);
+        document.getElementById("addPersonContainer").prepend(document.getElementsByClassName("pac-container")[0]);
     }, 300);
 }
+
+
+// Adding a new meetup attendee 
+function addPerson() {
+    // Create new attendee dict
+    var address = document.getElementById("address").value;
+    var transport = document.getElementById("transport").value;
+    var newAttendee = {
+        "address": address,
+        "transport": transport
+    };
+
+    // Add to attendees list 
+    attendees.push(newAttendee);
+
+    // Add marker to the map and recenter
+    getLatLng(newAttendee);
+
+    // Reset the form inputs
+    document.querySelector('#addPerson').reset();
+}
+
+
+// Finding the meetup 
+function findMeetup() {
+    if (!validateInput()) {
+        return;
+    }
+
+    maxTravelTime = document.getElementsByClassName('slider')[0].value;
+
+    drawPaths();
+
+    alert("Not Finished Yet!");
+}
+
+
+function validateInput() {
+    if (attendees.length == 0) {
+        alert("You must add at least one person");
+        return false;
+    } if (destinationType == null) {
+        alert("You must select a destination type");
+        return false;
+    }
+    
+    return true;
+}
+
+
+function drawPaths(){
+    var directionsService = new google.maps.DirectionsService();
+
+    for (var i = 0; i < attendees.length; i++) {
+        var request = {
+            origin: attendees[i].address,
+            destination: mapCenter,
+            travelMode: getTransitMode(attendees[i].transport)
+        };
+
+        var directionsRenderer = new google.maps.DirectionsRenderer();
+        directionsRenderer.setMap(map);
+        
+        directionsService.route(request, function(response, status) {
+            if (status === 'OK') {
+                // This is for later when rendering route duration in infowindows
+                // console.log(response.routes[0].legs[0].duration);
+
+                directionsRenderer = new google.maps.DirectionsRenderer();
+                directionsRenderer.setMap(map);
+                directionsRenderer.setDirections(response);
+            }
+        });
+    }
+}
+
+function getTransitMode(transport) {
+    if (transport == "car") {
+        return "DRIVING";
+    } else if (transport == "public") {
+        return "TRANSIT";
+    } else if (transport == "bike") {
+        return "BICYCLING";
+    } else if (transport == "walk") {
+        return "WALKING";
+    }
+}
+
+
+var latLngs = [];
+function getLatLng(attendee) {
```

Comment:
I would call this appendLatLng since you're adding onto an existing array. But maybe it's better to return the LatLng and then append it outside this function.